feature-libs/checkout/base/core/connectors/delivery-modes/converters.ts

```diff
@@ -0,0 +1,7 @@
+import { InjectionToken } from '@angular/core';
+import { DeliveryMode } from '@spartacus/core';
+import { Converter } from '@spartacus/core';
+
+export const DELIVERY_MODE_NORMALIZER = new InjectionToken<
+  Converter<any, DeliveryMode>
```

Comment:
I just copied / moved these files as is, but we can discuss on the approach we want to have. Please leave the comment unresolved, so that we can go back to it during a call